### PR TITLE
Adding a msc_type tier

### PR DESCRIPTION
**Pull request by marianne-m**
- [ ] tests
- [x] documentation
- [x] implementation

### CODE REVIEW

**Review by lucasgautheron — APPROVED** [review]

**Comment by lucasgautheron** [comment]
The tests fail because https://github.com/LAAC-LSCP/ChildProject/blob/master/tests/truth/eaf.csv lacks an empty msc_type column. One way to solve the problem is to add such a column to this csv.

Then it should be alright!

File: ChildProject/converters.py [hunk 1]
```diff
@@ -472,6 +472,7 @@ def convert(filename: str, filter=None) -> pd.DataFrame:
                     "vcm_type": "NA",
                     "lex_type": "NA",
                     "mwu_type": "NA",
+                    "msc_type": "NA",
```
Comment (lucasgautheron) [inline ChildProject/converters.py:475]: open question: should it be NA by default (in which case the tests need some adaptation), or simply undefined?

**Review by lucasgautheron — CHANGES_REQUESTED** [review]
Need to decide whether msc_type is defined to NA by default or undefined by default.
Tests will require a very minor fix depending on that choice.

File: ChildProject/converters.py (hunk 1 above)
Comment (marianne-m) [reply ChildProject/converters.py:475]: I'm not sure to understand the difference between the two.
Undefined means that the column will not appear if the "msc" tier isn't in the eaf file ?

Comment (lucasgautheron) [reply ChildProject/converters.py:475]: Yes, exactly!

Comment (marianne-m) [reply ChildProject/converters.py:475]: In that case I feel that the undefined way is better - the "msc_type" column doesn't exist in the already imported annotations

**Review by lucasgautheron — APPROVED** [review]

**Pull request merged**